tracim/development.ini.base

```diff
@@ -38,6 +38,9 @@ cache_dir = %(here)s/data
 beaker.session.key = tracim
 beaker.session.secret = 3283411b-1904-4554-b0e1-883863b53080
 
+# Auth type
+auth_type = internal
+
```

Comment:
Also : add ldap configuration block (commented) for users to know that ldap is ready to use even without to read the documentation